### ISSUE TITLE
Concourse pool resource should be configurable

### ISSUE THREAD

**Issue by jimmykarily** [cloudfoundry-incubator/kubecf#665]
**Is your feature request related to a problem? Please describe.**

The Concourse pipeline is using the [concourse pool resource](https://github.com/concourse/pool-resource#source-configuration) and is using the [cf-ci-pools](https://github.com/SUSE/cf-ci-pools) repository as a backend. People deploying their own instance of the pipeline should be able to point it to some repository they own.

**Describe the solution you'd like**
Being able to set the backend repository for the pool resource in the config.yaml file: https://github.com/cloudfoundry-incubator/kubecf/blob/master/.concourse/config.yaml.sample

**Comment by f0rmiga** [comment]
@fargozhu This is high priority as it blocks other contributors to KubeCF from deploying the pipeline.

**Comment by jimmykarily** [comment]
If we go the GKE way (https://github.com/cloudfoundry-incubator/kubecf/pull/859/files) this will no longer be relevant.

**Comment by mook-as** [comment]
Given that #859 is merged, I assume this is obsolete? I can't quite tell the state of things, though: I still see a resource named [`kind-environments`](https://concourse.suse.dev/teams/main/pipelines/kubecf/resources/kind-environments) in CI, but I don't see where it comes from in the [pipeline](https://github.com/cloudfoundry-incubator/kubecf/blob/master/.concourse/pipeline.yaml.gomplate). Doing a deployment of the pipeline (cancelled, of course) shows a lot of changes too.

**Comment by jimmykarily** [comment]
The PR was merged but the pipeline was not deployed (we tested the new version of the pipeline on a deployed "copy"). I deployed the kubecf pipeline now. Using gke for temporary cluster is a bit of an experiment at the moment. We may have to revert back to the old solution if gke isn't viable (cost wise, performance wise etc). I would keep this open for at least one more Sprint, to evaluate the new pipeline.
